client/parser/version.go

```diff
@@ -0,0 +1,25 @@
+package parser
+
+import (
+	"fmt"
+
+	"github.com/deis/deis/version"
+	docopt "github.com/docopt/docopt-go"
+)
+
+// Version displays the client version
+func Version(argv []string) error {
+	usage := `
+Displays the client version.
+
+Usage: deis version
+
+Use 'deis help [command]' to learn more.
+`
+	if _, err := docopt.Parse(usage, argv, true, "", false, true); err != nil {
```

Comment:
slight overkill but doesn't really do much, but provides a nice environment for future enhancements to this command like deis/workflow#329